src/arrayMethodSort.js

```diff
@@ -1,11 +1,34 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+const defaultCompare = (a, b) => {
+  if (String(a) > String(b)) {
+    return 1;
+  } else if (String(a) < String(b)) {
+    return -1;
+  }
+
+  return 0;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultCompare) {
+    let len = this.length * 2;
```

Comment:
Let's replace it with changes counter.
So, if previous run made no changes, we exit the loop.